blog-guidelines.md

```diff
@@ -2,7 +2,7 @@
 
 ## CDF Blog Overview ##
 
-The Continuous Delivery Foundation (CDF) blog serves as a channel for CDF members, project maintainers of graduated and incubating projects, and ambassadors to share content with the community. 
+The Continuous Delivery Foundation (CDF) blog serves as a channel for CDF members, project maintainers of graduated and incubating projects, and ambassadors to share content with the community. Please note: You need to be a CDF member, any tier, to publish through the CDF blog.
```

Comment:
Just noticed this addition and wanted to jump in with my usual reaction; I'm not sure if this passes the right message regarding what CDFoundation envisions to be - "A Neutral Home for the Next Generation
of Continuous Delivery Collaboration", the keyword being collaboration.

I think this type of messaging is kind of conflicting with what CDFoundation aims for - at least that's what I understand and expect the vision set by CDFoundation. If CDFoundation puts membership as a prerequisite for something like blog posts, how people can come, contribute, and collaborate with even more serious things such as projects they created?

Aside from that, the preceding sentence which was there before this PR does not feel right either. If CDFoundation aims to be the neutral home, it should encourage people to bring their ideas, articles, and so on to the foundation without asking them to be working for member companies.